discord/message.go

```diff
@@ -181,20 +181,26 @@ type Sticker struct {
 	// ID is the ID of the sticker.
 	ID StickerID `json:"id"`
 	// PackID is the ID of the pack the sticker is from.
-	PackID StickerPackID `json:"pack_id"`
+	PackID StickerPackID `json:"pack_id,omitempty"`
 	// Name is the name of the sticker.
 	Name string `json:"name"`
 	// Description is the description of the sticker.
 	Description string `json:"description"`
 	// Tags is a comma-delimited list of tags for the sticker. To get the list
 	// as a slice, use TagList.
 	Tags string `json:"-"`
```

Comment:
```suggestion
	Tags string `json:"tags"`
```
This wasn't subject to your PR, but this json tag should be set to the above. Otherwise, tags won't get (de-)serialized. It would be great if you could change this too, then we wouldn't need another PR just to fix this.